gradle/checkstyle.gradle

```diff
@@ -8,6 +8,6 @@ task("checkstyle", type: Checkstyle) {
     configFile rootProject.file("config/checkstyle/checkstyle.xml")
     source  "src/"
     include "**/*.java"
-    exclude "**/external/**/*.java"
+    exclude "**/external/**/*.java", "**/JsonReaderTest.java"
```

Comment:
Is this excluding the test added from the GSON codebase?

Reply:
Yep. We've excluded that test from checkstyle as GSON uses a different codestyle to us, so the checkstyle would fail the build otherwise.